### DISCUSSION TITLE
无效：无法复现

### DISCUSSION THREAD

**Discussion by linger040831** [hooke007/mpv_PlayKit#244] [回收]
uosc的顶栏字体好像不跟随[ script-opts-append = custom_font=](https://github.com/hooke007/MPV_lazy/blob/1fb2ad48c0a9b1bdcb43d4df1a7277346daf55ea/portable_config/script-opts.conf#L544) 这个的设置，请问是有单独的参数，还是不能更改字体？

**Comment by hooke007** [comment]
无法复现
![Snipaste_2022-12-13_14-23-56](https://user-images.githubusercontent.com/41094733/207358903-650bbe86-e485-43f3-907b-73481b284895.jpg)

**Reply by linger040831** [reply]
我把uosc换成https://github.com/tomasklaen/uosc这个原版的就正常了，已经解决了

**Reply by hooke007** [reply]
原版没有这个功能

**Comment by hooke007** [comment]
<s>暂时不锁话题，直至其它人反馈此功能是否正常</s>

**Reply by hooke007** [reply]
<img width="353" alt="截屏2022-12-13 16 55 23" src="https://user-images.githubusercontent.com/41094733/207395795-add48983-bbbb-425f-a255-f95006d541c6.png">
经测试mac端也正常
